monitoring/epaper_dashboard.py

```
@@ -0,0 +1,312 @@
+from datetime import datetime
+from waveshare_epd import epd2in13_V4
+from PIL import Image, ImageDraw, ImageFont
+
+# --- Connection config ---
+MARCEL_HOST = "marcel@10.67.0.2"
+MARCEL_KEY = "/home/cam/.ssh/finland_key"
+MARCEL_PORT = "2078"
+SYSTEM_API = "http://127.0.0.1:9191"
+SITE_API = "http://localhost:8393"
+API_USER = "gagan"
+API_PASS = "OPUyaW9jHeO3Fc3B1wV0gQl0!A1"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "API_PASS"

(S105)